sourcemash/frontend/static/js/views/forgot_modal_view.js

```diff
@@ -0,0 +1,35 @@
+Sourcemash.Views.ForgotModalView = Backbone.View.extend({
+    template: JST['forgot-modal'],
+
+    events: {
+        'submit #forgot-form': 'recoverFromModal',
+        'click .forgot-close': 'activateLoginForm'
+    },
+
+    recoverFromModal: function(e) {
+        e.preventDefault();
+        var formData = JSON.stringify($("#forgot-form").serializeObject());
+        var posting = $.ajax({
+                          type: "POST",
+                          url: "/reset",
+                          data: formData,
+                          success: this.recoverPassword,
+                          contentType: "application/json"
+                      });
+    },
+
+    recoverPassword: function(data) {
+      $("#forgot-modal").closeModal();
+      toast("Check your email for further instructions.", 3000);
+    },
+
+    activateLoginForm: function() {
+      toast("<-- Login over there!", 3000)
```

Comment:
Up to you. I think the modal looks a little nicer with a footer, and again it doesn't do any harm to leave it there. The reset password form is very small and flat, so these buttons add some form of interaction and interest, even if it's not strictly necessary.